src/classes/AnonymisePersonExtension.cls

```diff
@@ -17,7 +17,7 @@
  *  20130403    MT    Check that all service usages are not active.
  *                             
 *******************************************************************************************/
-public with sharing class AnonymisePersonExtension {
+public with sharing class Anonymise_PersonExtension {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.